### PR TITLE
minor replicator fixes

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,6 @@
     "got": "^9.6.0",
     "i18next": "^17.0.13",
     "jsonfile": "^5.0.0",
-    "mz": "^2.7.0",
```
Comment (manuel-calavera) [inline package.json]: this package outdated and there is native fs which is similar and better to use

File: src/replicator.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ export class Replicator {
         [
           `solana-install${dotExe}`,
           `solana-replicator${dotExe}`,
-          `solana${dotExe}`,
+          `solana-wallet${dotExe}`,
```
Comment (manuel-calavera) [inline src/replicator.js]: did we rename it to -wallet?

File: src/replicator.js [hunk 3]
```diff
@@ -241,9 +241,9 @@ export class Replicator {
 
       await this.cmd(solanaInstallInit, [
         '--config',
-        this.solanaInstallConfig,
+        `"${this.solanaInstallConfig}"`,
```
Comment (manuel-calavera) [inline src/replicator.js]: path has to be quoted otherwise it would not work if path contains space

File: src/replicator.js (hunk 2 above)
Comment (mvines) [reply src/replicator.js]: No, it's called `solana`

File: package.json (hunk 1 above)
Comment (mvines) [reply package.json]: native fs is not Promise aware right?  So you can't `await` on anything in `fs`

**Comment by mvines** [comment]
This is really two PRs please:
1) fs -> mz/fs
2) the solana cli rename

Small PRs that are about 1 thing are much easier to review and land

**Pull request closed without merging**